NOTICES

```diff
@@ -829,7 +829,7 @@ performance-now 2.1.0 : MIT License
 phadej/cabal-doctest 1.0.8 : BSD 3-clause "New" or "Revised" License
 picomatch 2.2.3 : MIT License
 picomatch 2.3.0 : MIT License
-pip 21.1.2 : MIT License
+pip 21.2.3 : MIT License
```

Comment:
same thing as yesterday, I don’t understand it but afaict it’s not more wrong than what we had before either so merging.